### PR TITLE
Fixed #5181 Misleading warning title for duplicate file name

### LINKED ISSUE #5181: [Bug]: Misleading warning title for duplicate file name
### Summary

When uploading a file with caption that causes a duplicate file name, the title of the warning says `Duplicate Image Found` which seems to imply that the image itself is duplicated instead of the file name.

### Steps to reproduce

1. Upload an image with the caption field `asd`

### Expected behaviour

`Duplicate File Name found` as title

### Actual behaviour

`Duplicate Image Found`

### Device name

Pixel XL

### Android version

API 30

### Commons app version

master

### Device logs

_No response_

### Screen-shots

image

### Would you like to work on the issue?

None

### PR DESCRIPTION
**Pull request by shankarpriyank**
**Description (required)**

Fixes #5181 

What changes did you make and why?
Added a new string resource to display in the alertdialog

**Tests performed (required)**

Tested locally on my device POCO X2, API Level 30

**Screenshots (for UI changes only)**

| Before | After |
| --- | --- |
|image | image

---

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -192,6 +192,7 @@
   <string name="location_permission_title">Requesting Location Permission</string>
   <string name="ok">OK</string>
   <string name="warning">Warning</string>
+  <string name="duplicate_file_name">Duplicate File Name found</string>
   <string name="duplicate_image_found">Duplicate Image Found</string>
```
Comment (nicolas-raoul) [inline app/src/main/res/values/strings.xml:196]: Is this duplicate_image_found still used somewhere?

Comment (shankarpriyank) [reply app/src/main/res/values/strings.xml:196]: Nope.

Comment (shankarpriyank) [reply app/src/main/res/values/strings.xml:196]: Seems like we can delete it

Comment (nicolas-raoul) [reply app/src/main/res/values/strings.xml:196]: Please remove it then 🙂

Comment (shankarpriyank) [reply app/src/main/res/values/strings.xml]: Done!

**Comment by shankarpriyank** [comment]
@nicolas-raoul I guess this pr is good to go?

**Review by jocoand — APPROVED** [review]

**Review by nicolas-raoul — APPROVED** [review]
Thanks and sorry for the delay!

**Pull request merged**